apps/meteor/app/slashcommands-status/client/status.ts

```diff
@@ -1,26 +1,27 @@
 import { Meteor } from 'meteor/meteor';
 import { TAPi18n } from 'meteor/rocketchat:tap-i18n';
-import type { IMessage } from '@rocket.chat/core-typings';
 
 import { slashCommands } from '../../utils/lib/slashCommand';
 import { settings } from '../../settings/server';
 import { api } from '../../../server/sdk/api';
 import { handleError } from '../../../client/lib/utils/handleError';
 
-function Status(_command: 'status', params: string, item: IMessage): void {
-	const userId = Meteor.userId() as string;
+slashCommands.add(
+	'status',
+	function Status(_command, params, item): void {
```

Comment:
Nesting this function declaration as a parameter expression damages readability, as evidenced by the indentation increase.